bezier/src/lib.rs

```diff
@@ -249,6 +260,32 @@ impl QuadraticBezierSegment {
     }
 }
 
+impl HasBoundingRect for QuadraticBezierSegment {
+    fn bounding_rect(&self) -> Rect {
+        let min_x = self.from.x.min(self.ctrl.x).min(self.to.x);
+        let max_x = self.from.x.max(self.ctrl.x).max(self.to.x);
+        let min_y = self.from.y.min(self.ctrl.y).min(self.to.y);
+        let max_y = self.from.y.max(self.ctrl.y).max(self.to.y);
+
+        return Rect::new(Point::new(min_x, min_y), Size::new(max_x - min_x, max_y - min_y));
```

Comment:
https://autumnai.github.io/cuticula/euclid/rect/struct.Rect.html I don't find any rect constructor which would allow me to do this, could you please point me to the right spot ?